app/src/features/projects/components/ProjectLocationForm.tsx

```diff
@@ -325,7 +325,7 @@ const ProjectLocationForm: React.FC<IProjectLocationFormProps> = (props) => {
               }
               key={index}
               onMouseEnter={() => mouseEnterListItem(index)}
-              onMouseLeave={() => mouseLeaveListItem(index)}>
+              onMouseLeave={() => mouseLeaveListItem()}>
```

Comment:
Instead of a basic DIV component, could we utilize or create a common component that handles the mask states and hovers?
This will also help code readability and usability for the location Form component.

Reply:
That's a good idea. Would it help for testing?
As for readability... I would disagree.

Reply:
I'm going to create a new ticket for this and put it in the next sprint.